### PR TITLE
Update DataPacket.php &&  Others

### PR DESCRIPTION
**Pull request by shoki-3738**

### CODE REVIEW

File: src/raklib/protocol/EncapsulatedPacket.php [hunk 1]
```diff
@@ -50,20 +50,22 @@ class EncapsulatedPacket{
 	 *
 	 * @return EncapsulatedPacket
 	 */
-	public static function fromBinary($binary, $internal = false, &$offset = null){
+	public static function fromBinary(&$binary, $internal = false, &$offset = null){
```
Comment (GabiYT) [inline src/raklib/protocol/EncapsulatedPacket.php]: for what is tehe "&"? :)

Comment (NycuRO) [inline src/raklib/protocol/EncapsulatedPacket.php]: @GabiYT Variable. Exist on php, but it's not ok there.

**Comment by DerddyDert** [comment]
You guys are awesome! @NycuRO @shoki-3738 @NycuRO

**Comment by DerddyDert** [comment]
@shoki-3738 PLEASE explain what the purpose of the PR is, what does it do, and did you test it? Thanks.

File: src/raklib/protocol/EncapsulatedPacket.php (hunk 1 above)
Comment (erkam2002) [inline src/raklib/protocol/EncapsulatedPacket.php]: LOL

**Comment by shoki-3738** [comment]
I have tested.
There was not any problem in particular.


This code was written by another developer in github
It seems to be a lightweight code
(by google translation)

**Comment by shoki-3738** [comment]
I think that Genisys Pro should be a little lighter src

**Comment by DerddyDert** [comment]
@shoki-3738  I agree.

**Comment by shoki-3738** [comment]
Leave a comment

**Pull request closed without merging**